openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts

```diff
@@ -579,60 +622,143 @@ test.describe('Teams Page', () => {
     await table1.create(apiContext);
     await team1.create(apiContext);
 
-    await addTeamOwnerToEntity(page, table1, team1);
-    await verifyAssetsInTeamsPage(page, table1, team1, 1);
+    await addTeamOwnerToEntity(adminPage, table1, team1);
+    await verifyAssetsInTeamsPage(adminPage, table1, team1, 1);
 
     // Navigate to users tab and add new user
-    await page.locator('[data-testid="users"]').click();
+    await adminPage.locator('[data-testid="users"]').click();
 
-    const fetchUsersResponse = page.waitForResponse(
+    const fetchUsersResponse = adminPage.waitForResponse(
       '/api/v1/users?limit=25&isBot=false'
     );
-    await page.locator('[data-testid="add-new-user"]').click();
+    await adminPage.locator('[data-testid="add-new-user"]').click();
     await fetchUsersResponse;
 
     // Search and select the user
-    await page
+    await adminPage
       .locator('[data-testid="selectable-list"] [data-testid="searchbar"]')
       .fill(user.getUserName());
 
-    await page
+    await adminPage
       .locator(
         `[data-testid="selectable-list"] [title="${user.getUserName()}"]`
       )
       .click();
 
     await expect(
-      page.locator(
+      adminPage.locator(
         `[data-testid="selectable-list"] [title="${user.getUserName()}"]`
       )
     ).toHaveClass(/active/);
 
-    const updateTeamResponse = page.waitForResponse('/api/v1/users*');
+    const updateTeamResponse = adminPage.waitForResponse('/api/v1/users*');
 
     // Update the team with the new user
-    await page.locator('[data-testid="selectable-list-update-btn"]').click();
+    await adminPage
+      .locator('[data-testid="selectable-list-update-btn"]')
+      .click();
     await updateTeamResponse;
 
     // Verify the user is added to the team
     await expect(
-      page.locator(`[data-row-key="${userName.toLowerCase()}"]`)
+      adminPage.locator(`[data-row-key="${userName.toLowerCase()}"]`)
     ).toBeVisible();
 
-    await page
+    await adminPage
       .locator(`[data-row-key="${userName.toLowerCase()}"]`)
       .getByTestId('remove-user-btn')
       .click();
 
-    const updatedTeamResponse = page.waitForResponse('api/v1/users*');
+    const updatedTeamResponse = adminPage.waitForResponse('api/v1/users*');
 
-    await page.getByRole('button', { name: 'confirm' }).click();
+    await adminPage.getByRole('button', { name: 'confirm' }).click();
     await updatedTeamResponse;
 
     await expect(
-      page.locator(`[data-row-key="${userName.toLowerCase()}"]`)
+      adminPage.locator(`[data-row-key="${userName.toLowerCase()}"]`)
     ).not.toBeVisible();
 
     await afterAction();
   });
 });
+
+test.describe('Teams Page with EditUser Permission', () => {
+  test.slow(true);
+
+  test.beforeAll('Setup pre-requests', async ({ browser }) => {
```

Comment:
it you are using different describe block for the permission, then all the above change from `page` to `adminPage` is not needed right? you can simply use below code in the describe block
```test.use({ storageState: { cookies: [], origins: [] } });```